### ISSUE TITLE
Fix peer review flagged issues

### ISSUE THREAD

**Issue by mannbiher** [mannbiher/DeepLearningForHealthCareProject#72]
- [ ] Up and downside of patching method
- [ ] In some cases, the "ground class opacity" can be seen in the edge of the lung area, which may be susceptible to be cropped by such methods. 
- [ ] if majority voting for the patches are better or other weighted averaging methods. 
- [ ] there were a few grammar mistakes, for instance:" Conclusion provided in the patch based paper [6] by Park and Ye, it is clear that..." 
- [x] Add discussion section 
- [ ] For pseudocode, make sure to indent for the "For each batch.." 
- [ ] more citations would be better. 
- [ ] Each of the predictions are then passed through another neural weight network"" - do you mean ""weighted neural network""
- [ ] How are the base classifiers of the FLANNEL ensemble different?
- [ ] Figure 1 is super helpful in putting it all together but could be arranged/labeled better.  
- [ ] The 'model predictions using majority voting' label should be moved out by the 'weighted ensemble' block.  
- [ ] Also the circular items in the final/real predictions area are strange.  I think it should be something more like what you see in an actual paper such as y_1 ... y_i and y hat_1 ... y_hat_i
- [x] I m not 100% clear on the distinction between 'Old data' and I assume 'new data'.  Is this a significant thing that needs explained?
- [ ] A lot of significant things that you would like to do.  I would caution against trying to do too many things.  
- [ ] Figure out the most important parts to finish up so that you have time left to 'polish' things.
- [x] Figure 3 should have its Y-margins adjusted.  Maybe 0.6 to 0.9. 
- [x] it would probably be better to also include the percentage of improvement in Result instead of in the conclusion.
- [x] more related works might need to be included so that the readers know the current state of the art models that attempt to solve this problem and how they perform.
- [ ] if Abstract should contain brief of all sections or just the paper abstract as an overview. 
- [x] Related Work if included as a separate section could be easier for readers to follow standardized reading format.
- [x] The paper seems to have no Discussion section where post experiments the experience and outcome are being discussed before the conclusion.
- [x] The number of overall images used for training, validation and test aren't clear and not mentioned in detail in the METHOD section. Also not mentioned out of the dataset, how many of them are covid-19 Chest X-rays  and how many of them are non-covid Chest X-rays
- [x] missed to include environment details like which environment they finalized to implement their code and test

**Comment by mannbiher** [comment]
Significant peer review issues are fixed.
